_data/changelogs/component-icon.yml

```diff
@@ -2,6 +2,13 @@ title: Icon
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
```

Comment:
**chore**: Update date before merge.

```suggestion
  - date: NNNN-NN-NN
```

Reply:
Resolved in [93d5546](https://github.com/uswds/uswds-site/pull/2951/commits/93d554676a65798a6458efa3e93e7b5441596278)